hack/update-estimator-protobuf.sh

```diff
@@ -36,10 +49,11 @@ ${GOPATH}/bin/go-to-protobuf \
 # The `go-to-protobuf` tool will modify all import proto files in vendor, so we should use go mod vendor to prevent.
 go mod vendor
 
-SERVICE_PROTO_FILES=$(find . -name "service.proto")
+SERVICE_PROTO_FILES=$(find . -not -path *./_tmp/* -name "service.proto")
```

Comment:
I guess if the proto has been copied into other directories, the path dependant would be wrong. The `protoc` cmd line is a little bit dumb.

Reply:
In my own machine, this work well, I guess whether this problem is related with code location.On my own machine, code is located in GOPATH directory, like `/root/go/src/github.com/karmada-io/karmada`, But on CI machine, it is not like this.

Reply:
I just had a try and still had some errors in my local environment. The error is like: 
```
root@VM-216-239-centos:~/go/src/github.com/karmada-io/karmada [⏱ 14:36:18]
.......
./pkg/estimator/pb/service.proto: Input is shadowed in the --proto_path by "/root/go/src/github.com/karmada-io/karmada/pkg/estimator/pb/service.proto".  Either use the latter file as your input or reorder the --proto_path so that the former file's location comes first.
```
My code is in $GOPATH, it's a little bit weird.

Reply:
It's due to KARMADA_ROOT this variable, after I delete this, it works well.
this is introduced today when I try to fix this problem.